warehouse/utils/wsgi.py

```diff
@@ -43,11 +50,14 @@ def __call__(self, environ, start_response):
             remote_addr = _forwarded_value(
                 environ.get("HTTP_X_FORWARDED_FOR", ""), self.num_proxies
             )
+            remote_addr_hashed = ""
```

Comment:
Should we be hashing `HTTP_X_FORWARDED_FOR` here instead?

Reply:
Good idea, will add!

Reply:
Added in f8d8061